examples/objectServerExample/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
     defaultConfig {
         applicationId 'io.realm.examples.objectserver'
         targetSdkVersion rootProject.sdkVersion
-        minSdkVersion 15
+        minSdkVersion 16
```

Comment:
Why update this to 16?

Reply:
because `android:fontFamily="sans-serif-light"` requires API level 16